### ISSUE TITLE
9.4 Develop 272 - Exhaustion setting should be not be in "play times"

### ISSUE THREAD

**Issue by MishkaFur** [StrangeLoopGames/EcoIssues#21696]
Exhaustion setting should not be in "play times" but should have its own option.

Would not obviously think to click into Play times to find exhaustion settings in the client

**Comment by IcedForge** [comment]
I agree this was very confusing to find need to get a better placement for this feature and i think its on the right track on giving it its own button for it
